src/test/java/org/openmrs/performance/http/DoctorHttpService.java

```diff
@@ -261,6 +261,16 @@ public HttpRequestActionBuilder getImmunizations(String patientUuid) {
 				.get("/openmrs/ws/fhir2/R4/Immunization?patient=" + patientUuid + "&_summary=data");
 	}
 	
+	public HttpRequestActionBuilder getPrograms() {
+		return http("Get Programs of Patient")
+				.get("/openmrs/ws/rest/v1/program?v=custom:(uuid,display,allWorkflows,concept:(uuid,display))");
+	}
+	
+	public HttpRequestActionBuilder getProgramEnrollment(String patientUuid) {
+		return http("Get Program Enrollment of Patient")
+				.get("/openmrs/ws/rest/v1/programenrollment?patient=" + patientUuid + "&v=custom:(uuid,display,program,dateEnrolled,dateCompleted,location:(uuid,display),states:(startDate,endDate,voided,state:(uuid,concept:(display))))");
+	}
```

Comment:
A method for this already exists, please reuse it.

[getProgramEnrollments](https://github.com/openmrs/openmrs-contrib-performance-test/blob/03d7d46a946a8cdd80c515b51c56ae1e09978168/src/test/java/org/openmrs/performance/http/DoctorHttpService.java#L87)